src/app/auth/login/page.tsx

```diff
@@ -29,10 +29,22 @@ const Login = () => {
     }
   };
 
+  const signInGoogle = async () => {
+    try {
+      await signIn('google');
+      toast('You have logged in successfully.', 'success');
+    } catch (err: any) {
+      toast(err.message, 'error');
+    }
+  };
+
   return (
     <main className="layout-auth">
       <h2 className="layout-auth__heading">Log In Form</h2>
       <LogInForm handleFormSubmit={handleFormSubmit} />
+      <button onClick={signInGoogle} className="form__button--submit">
```

Comment:
Thats an incorrect use of BEM. Classes must be as the following:

```suggestion
      <button onClick={signInGoogle} className="form__button form__button--submit">
```

The first class allows to inherit the general styles for `form__button` while the second class applies the variant styles for `form__button--submit`.

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/85/commits/4cf03096935de24b8d3cb3809ceb398658818661